scripts/migration/migrate_registration_extra.py

```diff
@@ -0,0 +1,56 @@
+"""
+Changes existing question.extra on Prereg registrations
+to a list. Required for multiple files attached to a question
+"""
+import sys
+import logging
+
+from modularodm import Q
+from website.app import init_app
+from scripts import utils as scripts_utils
+from website.models import Node
+from website.prereg.utils import get_prereg_schema
+from framework.transactions.context import TokuTransaction
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def migrate_file_meta(question):
+    files = question.get('extra')
+    if isinstance(files, dict):
+        if len(files) == 0:
+            question['extra'] = []
+        else:
+            question['extra'] = [files]
+
+
+def migrate():
+    PREREG_CHALLENGE_METASCHEMA = get_prereg_schema()
+    registrations = Node.find(
+        Q('is_registration', 'eq', True) &
+        Q('registered_schema', 'eq', PREREG_CHALLENGE_METASCHEMA) &
+        Q('is_deleted', 'eq', False)
```

Comment:
I was mostly thinking in terms of keeping the data consistent, including deleted nodes. If we were to recover a deleted node, it would break.

Could we just try-except if a registration has malformed data?